gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StatisticTypeEnum.java

```diff
@@ -26,5 +26,8 @@ public enum StatisticTypeEnum {
   TYPE_DURATION,
 
   /** Gauge. */
-  TYPE_GAUGE
+  TYPE_GAUGE,
+
+  /* Duration. Total duration for the execution of operation */
```

Comment:
Please add bit more context as to WHY we added this and how this is different from other StatisticTypes